driver/src/main/scala/api/collections/DeleteOps.scala

```diff
@@ -55,7 +55,7 @@ trait DeleteOps[P <: SerializationPack with Singleton]
     protected def bulkRecover: Option[Exception => Future[WriteResult]]
 
     /**
-     * Performs a [[https://docs.mongodb.com/manual/reference/method/db.collection.deleteOne/ single delete]] (see [[BatchCommands.DeleteCommand.DeleteElement]]).
+     * Performs a delete with a single selector (see [[BatchCommands.DeleteCommand.DeleteElement]]).
```

Comment:
removed the reference to deleteOne because this method can be used for both `deleteOne` and `deleteMany`